lib/polymorphic_integer_type/extensions.rb

```diff
@@ -64,8 +68,10 @@ def remove_type_and_establish_mapping(name, options, scope)
             condition = instance_exec(&scope).merge(condition) if scope.is_a?(Proc)
             condition
           }
+          return foreign_type, klass_mapping.to_i
```

Comment:
For readability sake, what do you think about adding a comment with an example of what we're expecting to return?

Reply:
like `source_type, 2`? but I doubt it will meant much for people without the full context of polymorphic setting (activeRecord, reflections and mapping)

Reply:
That's true! Never mind then!